go/bindings/build.go

```diff
@@ -139,7 +143,7 @@ func BuildCatalog(args BuildArgs) error {
 				log.WithField("request", request).Debug("capture validation requested")
 
 				var driver, err = args.CaptureDriverFn(ctx, request.EndpointType,
-					request.EndpointSpecJson, args.BuildAPI_Config.ConnectorNetwork, ops.StdLogger())
+					request.EndpointSpecJson, args.BuildAPI_Config.ConnectorNetwork, args.ContainerName, ops.StdLogger())
```

Comment:
The name of the capture being validated here is `request.Capture`. I'm thinking this is where it could call a function that accepts the task and operation names, and returns a container name to pass to `CaptureDriverFn`

Reply:
Patched below.